crosspm/cpm.py

```diff
@@ -66,9 +66,9 @@ def __init__(self):
             print(self._args)
             exit()
 
-        self.do_run(self.check_common_args)
+        # self.do_run(self.check_common_args)
 
-        self.do_run(self.read_config)
```

Comment:
You shouldn't just cut off initialization and do further testing. Let's move lines 69, 71 to run() method.